js/screens/ProfileScreen.js

```diff
@@ -0,0 +1,43 @@
+// @flow
+
+import React, { Component } from 'react'
+import { connect, type MapStateToProps } from 'react-redux'
+import { type NavigationScreenProp } from 'react-navigation'
+
+import { Container } from 'native-base'
+
+import type { Dispatch, AnyAction } from '../actions/types'
+import type { LoggedInUserState } from '../reducers/user'
+import ProfileComponent from '../components/ProfileComponent'
+
+type Props = {
+  user: LoggedInUserState,
+  navigation: NavigationScreenProp<*, AnyAction>,
+  dispatch: Dispatch
+}
+
+/**
+ * This screen show the profile to the authenticated user.
+ */
+class ProfileScreen extends Component<Props> {
+  render() {
+    const { user, navigation, dispatch } = this.props
+
+    return (
+      <Container>
+        <ProfileComponent
+          user={user}
+          navigation={navigation}
+          dispatch={dispatch}
+        />
+      </Container>
+    )
+  }
+}
+
+const mapStateToProps: MapStateToProps<*, *, *> = (state: Object) => ({
```

Comment:
don't use `Object`

Reply:
I am going to fix all the typing issues this in another PR that's quite finished,

Reply:
sorry I keep forgetting this, better leave TODOs around (even better TODOs that reference pivotal stories) so the reviewer know this is going to be fixed later

Reply:
The new PR add typing of the State, the Store and may other things so i have to add TODOs everywhere in the code, i prefer to avoid this. I am goin to add a note to the PR description to avoid flodding the code of TODOs. What do you think?

Reply:
yes that's fine too